cpp/mostDistant.cpp

```diff
@@ -0,0 +1,35 @@
+#include <iostream>
```

Comment:
thanks for all the contributions, for all future problems that you solve on the top of the file add a link to the problem like 
```cpp
// https://www.hackerrank.com/challenges/most-distant

#include <iostream>
...
```